app/src/androidTest/java/com/github/se/cyrcle/ui/report/AdminScreenTest.kt

```diff
@@ -156,4 +176,82 @@ class AdminScreenTest {
     composeTestRule.onNodeWithTag("ReportCard0").performClick()
     composeTestRule.onNodeWithTag("CheckReportsButton0").performClick()
   }
+
+  @Test
+  fun testAlertDialogAppearsOnMoreOptionsClick() {
```

Comment:
Using `assertIsDisplayed` might be a stronger assertion to consider over `assertExists`

Reply:
Indeed, forgot this